claat/cmd/export.go

```diff
@@ -62,14 +63,17 @@ func CmdExport(opts CmdExportOptions) {
 			ch <- &result{src, meta, err}
 		}(src)
 	}
+	errs := []error{}
 	for range args {
 		res := <-ch
 		if res.err != nil {
-			errorf(reportErr, res.src, res.err)
+			errs = append(errs, res.err)
+			log.Printf(reportErr, res.src, res.err)
 		} else if !isStdout(opts.Output) {
 			log.Printf(reportOk, res.meta.ID)
 		}
 	}
+	return len(errs) == 0
```

Comment:
Seems like this loses useful info - there's no way for the caller to find out details of the error(s).